gr-fec/python/fec/qa_fecapi_cc.py

```diff
@@ -191,6 +192,28 @@ def test_parallelism1_05(self):
 
         self.assertEqual(data_in, data_out)
 
+    def test_async_00(self):
+        frame_size = 30
+        k = 7
+        rate = 2
+        polys = [109, 79]
+        for mode in [fec.CC_TRUNCATED, fec.CC_TERMINATED, fec.CC_TAILBITING]:
```

Comment:
You may want to have a look at [unittest subtest](https://docs.python.org/3/library/unittest.html#distinguishing-test-iterations-using-subtests). This should help to have this test run with multiple parameters.

Reply:
I'm using subtest 3 lines below this. Is there a better way to use it? I've never known if there is a way to use subtest to generate different test *methods*. The usage I'm doing here (which is like the example in the docs) bugs me a little because it requires several `for`'s and levels of indentation to iterate over several parameters (I guess I could condense the nested `for`'s in a single `for` with `zip`).

Reply:
That seems to be the way to go. I didn't notice.